src/api/serializers.py

```diff
@@ -85,8 +100,6 @@ class JournalSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = journal_models.Journal
-        fields = ('pk', 'code', 'name', 'publisher', 'issn', 'description', 'current_issue', 'default_cover_image',
```

Comment:
Can you tell us why you removed this whole line?

Reply:
I've since put this line of code back. I plead ignorance + distraction, I thought you told me to delete it, so I did. Whoops! I put it right back and marked this conversation as resolved. On the off chance that GitHub really wanted me to answer your question, I'm answering it now.